### PR TITLE
=doc Consolidate configuration doc #1290

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #1290

### CODE REVIEW

**Comment by jlprat** [comment]
Both pages were identical so I put them in the same PR

File: docs/src/main/paradox/java/http/client-side/pool-overflow.md [hunk 1]
```diff
@@ -1,53 +0,0 @@
-# Pool overflow and the max-open-requests setting
-
-@ref[Request-Level Client-Side API](request-level.md) and @ref[Host-Level Client-Side API](host-level.md)
-use a connection pool underneath. The connection pool will open a limited number of concurrent connections to one host
-(see the `akka.http.host-connection-pool.max-connections` setting). This will limit the rate of requests a pool
-to a single host can handle.
-
-When you use the @ref[stream-based host-level API](host-level.md#using-the-host-level-api-in-a-streaming-fashion)
-stream semantics prevent that the pool is overloaded with requests. On the other side, when a new request is pushed either using
-`Http.singleRequest()` or when materializing too many streams using the same `Http().cachedHostConnectionPool`, requests
```
Comment (ktoso) [inline docs/src/main/paradox/java/http/client-side/pool-overflow.md]: huh these were wrong in java it seems, should have been `Http.get(system)`, could you fix that now in the scala file with the `@java` syntax?

Comment (jlprat) [inline docs/src/main/paradox/java/http/client-side/pool-overflow.md]: Sure, I usually did a quick read after merge, but this one completely went under my radar.

Comment (jlprat) [inline docs/src/main/paradox/java/http/client-side/pool-overflow.md]: Actually even Scala ones were wrong, `Http.singleRequest()` doesn't work. Fixed that.

File: docs/src/main/paradox/scala/http/client-side/pool-overflow.md [hunk 2]
```diff
@@ -7,8 +7,9 @@ to a single host can handle.
...
 When you use the @ref[stream-based host-level API](host-level.md#using-the-host-level-api-in-a-streaming-fashion)
 stream semantics prevent that the pool is overloaded with requests. On the other side, when a new request is pushed either using
-`Http.singleRequest()` or when materializing too many streams using the same `Http().cachedHostConnectionPool`, requests
-may start to queue up when the rate of new requests is greater than the rate that the pool can process requests.
+@scala[`Http().singleRequest()`]@java[`Http.get(system).singleRequest()`] or when materializing too many streams using the same
+@scala[`Http().cachedHostConnectionPool`]@java[`Http.get(system).cachedHostConnectionPool`], requests may start to queue
+up when the rate of new requests is greater than the rate that the pool can process requests.
 
 In such a situation `max-open-requests` per host connection pool will be queued to buffer short-term peaks of requests.
 Further requests will fail immediately with a `BufferOverflowException` with a message like this:
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/client-side/pool-overflow.md]: Below on this page there's a `#connection-level-api` reference which can be removed.

File: docs/src/main/paradox/scala/http/client-side/pool-overflow.md [hunk 3]
```diff
@@ -7,8 +7,9 @@ to a single host can handle.
 
 When you use the @ref[stream-based host-level API](host-level.md#using-the-host-level-api-in-a-streaming-fashion)
 stream semantics prevent that the pool is overloaded with requests. On the other side, when a new request is pushed either using
-`Http.singleRequest()` or when materializing too many streams using the same `Http().cachedHostConnectionPool`, requests
-may start to queue up when the rate of new requests is greater than the rate that the pool can process requests.
+@scala[`Http().singleRequest()`]@java[`Http.get(system).singleRequest()`] or when materializing too many streams using the same
+@scala[`Http().cachedHostConnectionPool`]@java[`Http.get(system).cachedHostConnectionPool`], requests may start to queue
+up when the rate of new requests is greater than the rate that the pool can process requests.
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/client-side/pool-overflow.md]: `... is greater than the rate at which the pool can process requests`?

Comment (jlprat) [inline docs/src/main/paradox/scala/http/client-side/pool-overflow.md]: sounds better

File: docs/src/main/paradox/scala/http/client-side/pool-overflow.md (hunk 2 above)
Comment (jlprat) [inline docs/src/main/paradox/scala/http/client-side/pool-overflow.md]: Right

**Comment by jonas** [comment]
Double approved by me, does that count as two 👍 ? ;)
Need this merged so I can update #1350.

**Comment by ktoso** [comment]
I approve too :-)

**Pull request merged**